lib/order.rb

```diff
@@ -0,0 +1,55 @@
+class Order
+    attr_reader :id
+    attr_accessor :products, :customer, :fulfillment_status
+
+    def initialize(id, products, customer, fulfillment_status = :pending)
+        @id = id
+        @products = products
+        @customer = customer
+        @fulfillment_status = fulfillment_status
+        # If invalid fulfillment status is given, raise ArgumentError
+        possible_status = [:pending, :paid, :processing, :shipped, :complete]
```

Comment:
possibles_status would make a good constant variable assigned at the beginning of your script.